api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -342,6 +342,15 @@ message ExternalProcessor {
   // Since ``request_header_mode`` is not applicable in any way, it's ignored in comparison.
   repeated ProcessingMode allowed_override_modes = 22;
 
+  // Decorator to introduce custom logic that runs after the ``ProcessingRequest`` is constructed, but
+  // before it is sent to the External Processor. The ``ProcessingRequest`` may be modified.
+  //
+  // .. note::
+  //   Processing request modifiers are currently in alpha.
+  //
```

Comment:
Can you please add an extension category, similar to [here](https://github.com/envoyproxy/envoy/blob/main/api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto#L351)?

Reply:
I think you'll need to introduce at least one actual implementation in this case.
The goal is to have for each extension point at least one concrete use-case to reduce dead-extension-points if possible, and to avoid potential future breakages.
Specifically I think that your "test" extension can be a non-test extension.